proto/btc/get_public_key.proto

```diff
@@ -20,11 +19,11 @@ message GetPublicKeyIntiateRequest {
   /*
    * address: m/44'/0'/0'/0/0
    */
-  repeated common.DerivationPath derivation_path = 2;
```

Comment:
If we are not using derivation path struct in xpub and public key, then it should be removed from common.proto file

Reply:
yes. eventually it will be removed

Reply:
it is still used at other places. are you suggesting to remove `common.DerivationPath` completely?

Reply:
If we agree that derivation path will not be required at other places as well, then yes, we should remove it now as that would mean less back and forth with SDK every we update this.

Reply:
okay. will take that change (remove `common.DerivationPath` completely) one go in the next PR.

Reply:
https://github.com/Cypherock/cypherock-common/pull/22

Reply:
:)